webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala

```diff
@@ -3585,4 +3585,106 @@ class OntologyV2R2RSpec extends R2RSpec {
       freetestLastModDate = responseAsInput.ontologyMetadata.lastModificationDate.get
     }
   }
+
+  "not create a property with invalid gui attribute" in {
+    val params =
```

Comment:
I'm not sure... but could this request be generated using the ontologyModels? would make the test slimmer.

Reply:
I followed the pattern of the other tests. Since I want to finish this PR before my holidays, I am not going to change it now but I would be happy to do that another time!